sematic/db/models/run.py

```diff
@@ -143,21 +148,21 @@ def validate_future_state(self, key, value) -> str:
             raise ValueError("future_state must be a FutureState, got {}".format(value))
 
     @validates("tags")
-    def convert_tags_to_json(self, key, value) -> str:
+    def convert_tags_to_json(self, _, value) -> str:
         if isinstance(value, list):
             return json.dumps(value)
 
         return value
 
     @validates("description")
-    def strip_description(self, key, value) -> str:
+    def strip_description(self, _, value) -> str:
         if value is not None:
             value = re.sub(r"\n\s{4}", "\n", value.strip())
 
         return value
 
     @property
-    def exception(self) -> Optional[ExceptionMetadata]:
+    def exception_metadata(self) -> Optional[ExceptionMetadata]:
```

Comment:
Let's keep this consistent with the DB field names.

Reply:
So either we revert to `exception`, or we rename the DB fields as well.

Reply:
Renaming `exception_json` to `exception_metadata_json` and `external_exception_json` to `external_exception_metadata_json` in the database, unless @augray has other suggestions.

Reply:
that's fine. Please make a note in `versions.py` that we will need to bump the min supported client version on the next release

Reply:
Renamed & added TODO in `versions.py`.